src/encoded/static/components/regulome_search.js

```diff
@@ -549,10 +549,11 @@ class FacetButton extends React.Component {
     }
 
     render() {
-        const { buttonLabel, lookupFilterCount, selectedFacets, facetLabel } = this.props;
+        const { buttonLabel, lookupFilterCount, selectedFacets, facetLabel, key } = this.props;
         const filterCount = lookupFilterCount(buttonLabel, facetLabel);
         return (
             <button
+                key={key}
```

Comment:
I don’t think you need to have the key here since this isn’t immediately in a loop. Just adding the key below in `Facet` like you have now should be fine.